src/System.Runtime.Extensions/tests/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
@josguil My comment was about your change in this PR.  You were changing the locked field from 'true' to 'false'.  You need to leave it as 'true'.  In general, once the project.lock.json file is made, the lock field should be set to 'true'.

If you look at other project.lock.json files in the CoreFx repo, you'll see that the locked field is set to 'true'.